invenio_communities/links.py

```diff
@@ -61,3 +61,17 @@ def default_links_pagination_factory(page, urlkwargs):
                                 **urlkwargs)
 
     return links
+
+def default_links_facets_factory():
```

Comment:
@blixhavn it is used with the endpoint definition in the api.py file inside view_func.